### PR TITLE
Add tutorial type and data

### PR DESCRIPTION
**Pull request by liorama**

### CODE REVIEW

File: scripts/src/create.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ import { Order } from "./models/orders";
 
 import { init as initModels } from "./models";
 import { getConfig } from "./config";
-import { animalPoll, kikPoll, Poll } from "./services/offer_contents";
+import { animalPoll, kikPoll, tutorial, Poll, Tutorial } from "./services/offer_contents";
```
Comment (doodyparizada) [inline scripts/src/create.ts]: optimize imports so `tutorial` would disappear

File: scripts/src/create.ts [hunk 2]
```diff
@@ -72,6 +72,10 @@ async function createOffers(): Promise<Offer[]> {
 		return offer;
 	}
 
+	offers.push(await createEarn("Dunkin Donuts", "Getting Started", "Learn about Kin",
```
Comment (doodyparizada) [inline scripts/src/create.ts]: Put `"Kin"` as a brand

Comment (doodyparizada) [reply scripts/src/create.ts]: also ask @noakessler for an image

File: scripts/src/mock_client.ts [hunk 3]
```diff
@@ -149,9 +149,59 @@ async function earnFlow() {
...
+		console.log("order still pending :(");
+	}
+
+	console.log(`got order after submit ${JSON.stringify(order, null, 2)}`);
+	console.log(`order history ${JSON.stringify((await client.getOrders()).orders.slice(0, 2), null, 2)}`);
+}
+
 async function main() {
-	await earnFlow();
-	await didNotApproveTOS();
```
Comment (doodyparizada) [inline scripts/src/mock_client.ts]: bring these back.

File: scripts/src/services/offer_contents.ts [hunk 4]
```diff
@@ -40,29 +51,58 @@ export const animalPoll: Poll = {
...
+	// const poll: Poll = JSON.parse(offer.content);
+	// // go over poll data, look for questions and check that answer is within question options
+	// for (const page of poll.pages) {
+	// 	const qId = page.question.id;
+	// 	const ans = page.question.answers;
+	//
+	// 	if (!ans.includes(parsed[qId])) {
+	// 		return false;
+	// 	}
+	// }
```
Comment (doodyparizada) [inline scripts/src/services/offer_contents.ts]: just delete it, you can retrieve it later from history

File: scripts/src/create.ts (hunk 1 above)
Comment (liorama) [reply scripts/src/create.ts]: `tutorial` is the data, the instance, and `Tutorial` is the type.

Comment (liorama) [reply scripts/src/create.ts]: I change `tutorial` to `getStartedTutorial` if it helps

**Pull request merged**